readme.md

```diff
@@ -51,7 +52,7 @@ const Root = () => (
 
 <a id="usetheatom" href="#usetheatom"><img src="img/doc.03.svg" alt="Use the atom in your components" /></a>
 
-It can be used just like `React.useState`:
+It can be used like React hooks (e.g. `React.useState`):
```

Comment:
Please revert this or suggest something else. We'd like make it specific to `useState`.